cmd/terramate/e2etests/list_test.go

```diff
@@ -95,6 +96,41 @@ func listTestcases() []testcase {
 				Stdout: "1\n2\n3\n",
 			},
 		},
+		{
+			name: "stack inside other stack",
+			layout: []string{
+				"s:stack",
+				"s:stack/child-stack",
+			},
+			want: runExpected{
+				Stdout: "stack\nstack/child-stack\n",
+			},
+		},
+		{
+			name: "multiple levels of stacks inside stacks",
+			layout: []string{
+				"s:mineiros.io",
+				"s:mineiros.io/departments",
+				"s:mineiros.io/departments/engineering",
+				"s:mineiros.io/departments/accounting",
+				"s:mineiros.io/departments/engineering/terramate",
+				"s:mineiros.io/departments/engineering/terraform-modules",
+				"d:mineiros.io/departments/engineering/docs",
+				"d:mineiros.io/departments/engineering/tests",
+				"s:mineiros.io/departments/engineering/tests/e2e",
+			},
+			want: runExpected{
+				Stdout: strings.Join([]string{
```

Comment:
Noice :100: , we could even have a stackList func or something that always does the joining with \n and accepts a vargs, so it would look like:

```go
Stdout: stackList(
   "stack1",
   "stack2",
)
```

Reply:
yep I thought about this, but introducing this make sense update all tests. I can do this in a separate PR.